examples/json-csv/json_csv.bal

```diff
@@ -0,0 +1,73 @@
+import ballerina/io;
+import ballerina/log;
+
+// Converts a json into an array of strings
```

Comment:
```suggestion
// Converts a `JSON` value to an array of strings.
```

Reply:
Some of the suggestions were addressed in #12858